packages/mdc-tooltip/constants.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** @enum {string} */
+const cssClasses = {
+  ROOT: 'mdc-tooltip',
+  SHOW: 'mdc-tooltip--show',
+  HIDE: 'mdc-tooltip--hide',
```

Comment:
instead of having a --show and a --hide class, can we default to hiding and remove the --hide class?

Reply:
The issue I had with that, was that the tooltips where just blinking up briefly on page load. 
I did change it to only having a show class and as a workaround I added an "animation" class which adds the animation for the default mode (hidden mode) after the first show animation. 
If you have a more elegant solution to this, let me know.

Reply:
I see - that sounds reasonable.